src/components/ConfirmationPage/ConfirmationPage.js

```diff
@@ -0,0 +1,298 @@
+import React from "react";
+import { Route } from "react-router-dom";
+import {
+	Box,
+	Heading,
+	Flex,
+	Text,
+	Link,
+	Icon,
+	Select,
+	Badge,
+	Checkbox,
+	Tooltip
+} from "@chakra-ui/core";
+import Helmet from "react-helmet";
+import Footer from "../Footer.jsx";
+import ValidatorTile from "../SuggestedValidators/ValidatorTile";
+import {
+	textColor,
+	textColorLight,
+	border,
+	getRiskLevelColor,
+	getRiskLevel
+} from "../../constants";
+import CustomButton from "../CustomButton";
+
+type ConfirmationPageProps = {
+	colorMode?: "light" | "dark",
+	stashOptions: Array<{ option: string, value: string }>,
+	controllerOptions: Array<{ option: string, value: string }>,
+	riskPreference: string,
+	validatorsList: Array<{
+		name: string,
+		avatar?: string,
+		amount: float,
+		risk?: float
+	}>,
+	eras: Number,
+	amount: float,
+	currency: string
+};
+
+const messageBoxColors = {
+	light: {
+		bg: "#E0F4FF",
+		border: "#C1E9FF",
+		text: "#89BEDC"
+	},
+	dark: {
+		bg: "#354056",
+		border: "#678AD1",
+		text: "#CCDDFF"
+	}
+};
+
+const ConfirmationPage = (props: ConfirmationPageProps) => {
+	const mode = props.colorMode ? props.colorMode : "light";
+
+	const [termsCheck, setTermsCheck] = React.useState(false);
+
+	return (
+		<>
+			<Helmet>
+				<title>Yield Scan &middot; Confirmation</title>
+			</Helmet>
+			<Route exact path='/confirmation'>
+				<Box m={4} mt={10}>
+					<Link m={4}>
+						<Icon name='arrow-back' mr={1} /> Back
+					</Link>
+				</Box>
+				<Flex w='100%' justify='center'>
+					<Box w={["100%", "80%", "60%", "50%"]}>
+						<Heading
+							as='h3'
+							size='xl'
+							textAlign='center'
+							color={textColor[mode]}
+						>
+							Confirmation
+						</Heading>
+						<Text
+							size='sm'
+							color={textColorLight[mode]}
+							textAlign='center'
+							w='100%'
+							height='auto'
+							mb={8}
+						>
+							You are about to stake your KSM on the following validators.
```

Comment:
Can we have global variable instead of hard-corded "KSM"?

Reply:
Yes, need to replace this with the currency prop. Thanks for catching this!